vmdb/app/models/ems_microsoft.rb

```diff
@@ -44,10 +44,16 @@ def verify_credentials(_auth_type = nil, options = {})
     raise MiqException::MiqHostError, "No credentials defined" if self.authentication_invalid?(options[:auth_type])
 
     begin
-      run_dos_command("hostname")
+      run_dos_command("powershell import-module virtualmachinemanager; " \
+      "Get-SCVMMServer -ComputerName localhost")
```

Comment:
No need for the line continuations for Strings.  Also indent the second line so it's clear it doesn't stand on its own.  Same goes for line 54-55 below.